modules/_labs/drawer/react/lib/DrawerHeader.tsx

```diff
@@ -12,12 +12,12 @@ export interface DrawerHeaderProps extends React.HTMLAttributes<HTMLDivElement>
   /**
    * The function called when the DrawerHeader close button is clicked.
    */
-  onClose: IconButtonProps['onClick'];
+  onClose?: IconButtonProps['onClick'];
   /**
    * The `aria-label` for the DrawHeader close button. Useful for i18n.
    * @default Close
    */
-  closeIconLabel: string;
+  closeIconLabel?: string;
```

Comment:
Just noticed a bug with the default for this. It's using `iconLabel` instead of `closeIconLabel`. Do you mind updating that as well?